### PR TITLE
Resume with LESS

### PR DESCRIPTION
**Pull request by rodpa715**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,153 @@
...
+              <section>
+                <h4>HTML & CSS</h4>
+                <ul>
+                  <li class="full"></li>
+                  <li class="full"></li>
+                  <li class="full"></li>
+                  <li class=""></li>
+                  <li class=""></li>
+                  <li class=""></li>
+                  <li class=""></li>
```
Comment (GianBarGian) [inline index.html]: Don't forget to clean up your code. You could want to remove these empty classes.

File: less/index.less [hunk 2]
```diff
@@ -1,59 +1,203 @@
...
-table {
-	border-collapse: collapse;
-	border-spacing: 0;
-}
+@import "./reset.less";
+@import "./font-prop.less";
+@import "./colors.less";
+@import "mixin.less";
+@import "./general.less";
+@import url("https://fonts.googleapis.com/css?family=Ubuntu:400,700");
```
Comment (GianBarGian) [inline less/index.less]: Good job on structuring your project in several LESS modules and importing them!

File: less/index.less [hunk 3]
```diff
@@ -1,59 +1,203 @@
...
-html, body {
-    height: 100%;
-}
+//breakpoint
+@mobile: 600px;
 
+/* CSS */
+.container {
+  margin: 0 auto;
+  max-width: 750px;
```
Comment (GianBarGian) [inline less/index.less]: You don't need to stick everything in you container here. It leads to more cluttered code.

Generally your aim is to divide your code in small chunks with not too much nesting. As a general rule of thumb you don't want to have more then 3 levels of nesting and maybe to don't have to scroll all the way up to see what are the specificity rules for that particular class you need to use. 
Basically small reusable chunks of easy to read/test/maintain code.

To sum up nesting here is kind of ok, but this huge class is way too long.

File: less/mixin.less [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.displayFlex(@f-dir){
+  display: flex;
```
Comment (GianBarGian) [inline less/mixin.less]: Good job with mixins!

**Comment by GianBarGian** [comment]
Very good job with the project Patrick. Your website looks awesome!

I really love the skills section, both from a design point of view and how you handled it.
Good job on breaking your LESS code in different modules.

Well done!
